CVs/input4MIPs_source_id.json

```diff
@@ -353,6 +353,27 @@
         "mip_era":"CMIP7",
         "source_version":"2.0"
     },
+    "FZJ-1-0":{
+        "activity_id": "input4MIPs",
+        "authors": [
+            {
+                "name": "Michaela Hegglin",
+                "email": "m.i.hegglin@fz-juelich.de",
+                "affiliations": [
+                    "Institute of Energy and Climate Research – Stratosphere (IEK-7), Forschungszentrum Jülich GmbH, 52425 Jülich, Germany",
+                    "Department of Meteorology, University of Reading, Earley Gate, Reading, RG6 6BB, United Kingdom"
+                ],
+                "orcid": "0000-0003-2820-9044"
+            }
+        ],
+        "contact":"Michaela Hegglin (m.i.hegglin@fz-juelich.de)",
+        "dataset_category":["Ozone", "SurfaceFluxes"],
+        "further_info_url":"https://input4mips-cvs.readthedocs.io/en/latest/dataset-overviews/nitrogen-deposition/",
```

Comment:
We also need to list ozone here - she's on the hook for two datasets

Reply:
@durack1 is going to follow up based on conversation we had